src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -571,6 +577,63 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 				});
 				widget.ariaLabel = ariaLabel;
 				widget.tabIndex = 0;
+			} else if (attachment.isImage) {
+				const blob = new Blob([attachment.value as Uint8Array], { type: 'image/png' });
+				const url = URL.createObjectURL(blob);
+
+				const img = document.createElement('img');
+				img.classList.add('chat-attached-context-image');
+				img.src = url;
+				img.alt = 'Image from Uint8Array';
+
+				// Pill with tiny image
+				const pill = document.createElement('div');
+				pill.classList.add('chat-attached-context-pill');
+				pill.tabIndex = -1;
+
+				const pillImg = document.createElement('img');
+				pillImg.src = url;
+				pillImg.alt = 'Image description';
```

Comment:
This alt text is actually worse than nothing, if we can't describe the image it's best to leave this as `alt=""` so the screen reader treats it as decorative